models/cloudtemplate_bgp_ipv4.go

```diff
@@ -0,0 +1,87 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/container"
+)
+
+const (
+	DncloudtemplateBgpIpv4        = "uni/tn-%s/infranetwork-%s/extnetwork-%s/vpnnetwork-%s/ipsec-[%s]/bgpipv4-[%s]"
+	RncloudtemplateBgpIpv4        = "bgpipv4-[%s]"
+	ParentDncloudtemplateBgpIpv4  = "uni/tn-%s/infranetwork-%s/extnetwork-%s/vpnnetwork-%s/ipsec-[%s]"
+	Cloudtemplatebgpipv4ClassName = "cloudtemplateBgpIpv4"
+)
+
+type BGPIPv4Peer struct {
```

Comment:
Would add CloudTemplateBGPIPv4Peer to differentiate. Same for other struct and name below.